src/sbt-test/sbt-conductr/sandbox-conductr-roles/build.sbt

```diff
@@ -21,25 +21,24 @@ BundleKeys.roles := Set("bundle-role-1", "bundle-role-2")
 
 val checkConductrRolesByBundle = taskKey[Unit]("Check that the bundle roles are used if no sandbox roles are specified.")
 checkConductrRolesByBundle := {
-  for (i <- 0 to 2) {
-    val content = s"docker inspect --format='{{.Config.Env}}' cond-$i".!!
-    val expectedContent = "CONDUCTR_ROLES=bundle-role-1,bundle-role-2"
-    content should not include(expectedContent)
+  val psOutput = s"ps ax".lines_!.toList
```

Comment:
We were going to introduce new `conduct` CLI commands in order to display the member/agent info, so I don't think that the `sandbox ps` command itself needs updating: https://github.com/typesafehub/conductr-cli/issues/294

We could leverage that work once done, but I don't think it is essential to do here given that we're talking about a test which'll always be running on some flavour of unix.